### PR TITLE
Network mar

### PR DESCRIPTION
**Pull request by khannarheams**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json [hunk 1]
```diff
@@ -0,0 +1,2553 @@
...
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of the resource."
+        }
+      },
+      "allOf": [
+
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json]: SSDL -> SSL?

**Comment by dsgouda** [comment]
@khannarheams 
- Please target the appropriate branch
@MikhailTryakhov Please take a look

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+            "BeginsWith",
+            "EndsWith",
+            "Regex"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallOperator",
+            "modelAsString": true
+          }
+        },
+        "negationConditon": {
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json]:
```suggestion
        "negationCondition": {
```

**Review by MikhailTryakhov — CHANGES_REQUESTED** [review]
1) please follow our process [https://microsoft.sharepoint.com/teams/WAG/AzureNetworking/_layouts/OneNote.aspx?id=%2Fteams%2FWAG%2FAzureNetworking%2FOneNotes%2FNRP%20migrated&wd=target%28Hydra%20%2B%20SDK.one%7C96F7B95B-A378-4CA5-903B-C88F894873EF%2FSwagger%20-%3E%20SDK%20-%3E%20PowerShell%20Process%20Overview%7C71B816ED-DAE5-4907-94D8-DE7EE13278A0%2F%29](One Note)

2) please target to ```network-march-release```
3) please DON'T MAKE functional changes in older API? We're not sure all of them may be supported on the server side
4) azure firewall just removed "Alert" from swagger, why are you returning this?

**Pull request closed without merging**